### ISSUE TITLE
OTX 2.0 fails to evaluate the exported models trained with tiling

### ISSUE THREAD

**Issue by goodsong81** [open-edge-platform/training_extensions#3166]
**Describe the bug**

```colsole
Traceback (most recent call last):
  File "/home/songkich/miniconda3/envs/otx-v2/bin/otx", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/mnt/sdb/workarea/otx/src/otx/cli/__init__.py", line 17, in main
    OTXCLI()
  File "/mnt/sdb/workarea/otx/src/otx/cli/cli.py", line 56, in __init__
    self.run()
  File "/mnt/sdb/workarea/otx/src/otx/cli/cli.py", line 570, in run
    self.instantiate_classes()
  File "/mnt/sdb/workarea/otx/src/otx/cli/cli.py", line 354, in instantiate_classes
    self.model, self.optimizer, self.scheduler = self.instantiate_model(model_config=model_config)
                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/sdb/workarea/otx/src/otx/cli/cli.py", line 443, in instantiate_model
    raise AttributeError(msg)
AttributeError: The model does not have a tile_config attribute. Please check if the model supports tiling.
```

**Steps to Reproduce**

```colsole
otx train --config src/otx/recipe/instance_segmentation/maskrcnn_efficientnetb2b_tile.yaml --data_root tests/assets/car_tree_bug/ --work_dir /tmp/tile --metric otx.core.metrics.fmeasure.FMeasure --callback_monitor val/f1-score --scheduler.monitor val/f1-score --seed 0 --deterministic False --max_epochs 1
otx export --work_dir /tmp/tile/
otx test --work_dir /tmp/tile/ --config src/otx/recipe/instance_segmentation/openvino_model.yaml --checkpoint /tmp/tile/.latest/export/exported_model.xml
```

**Comment by harimkang** [comment]
@eugene123tw Probably `tile_config.enable_tile` is on if we are using `openvino_model.yaml`.
We want `enable_tile` to be off in this case, right? 

Anyway,
In this case, we can use `otx test --work_dir /tmp/tile/ --checkpoint /tmp/tile/.latest/export/exported_model.xml`

But I think the example above should work fine. Can you take a look at the above case to make it work?

**Comment by eugene123tw** [comment]
PR opened: https://github.com/openvinotoolkit/training_extensions/pull/3204

**Comment by harimkang** [comment]
https://github.com/openvinotoolkit/training_extensions/pull/3229 Merged into 2.0
